### PR TITLE
Perf update notes from Prash

### PR DESCRIPTION
**Pull request by davidchesnut**
These are changes Prash introduced in the original repo under the following PRs
https://github.com/OfficeDev/VBA-content/pull/487
https://github.com/OfficeDev/VBA-content/pull/486

### CODE REVIEW

**Comment by Linda-Editor** [comment]
@davidchesnut @lindalu-MSFT We supposedly moved all the PRs off vba-content and updated topics accordingly, but I don't see these changes, so I'll go ahead and merge this PR.

File: excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ author: FastExcel
 
 Excel 2016 introduces new features that you can use to improve performance when you're working with large or complex Excel workbooks
 
+## VLOOKUP, HLOOKUP, MATCH improvements
+In Excel 16.0.9126.2xxx onwards VLOOKUP, HLOOKUP and MATCH for exact match on unsorted data is much faster than ever before when looking up multiple columns (or rows with HLOOKUP) from the same table range.
```
Comment (PrashSMSFT) [inline excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md]: Please change "In Excel 16.0.9126.2xxx" to "In Excel version 1809"

Comment (PrashSMSFT) [reply excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md]: Correction "In O365 version 1809 onwards, Excel's"

File: excel/Concepts/Excel-Performance/excel-tips-for-optimizing-performance-obstructions.md [hunk 2]
```diff
@@ -116,7 +116,9 @@ Dynamic ranges have the following advantages and disadvantages:
 
 ## Improve lookup calculation time
 
-Lookups are frequently significant calculation obstructions. Fortunately, there are many ways of improving lookup calculation time. If you use the exact match option, the calculation time for the function is proportional to the number of cells scanned before a match is found. For lookups over large ranges, this time can be significant.
+In Excel 16.0.9126.2xxx onwards VLOOKUP, HLOOKUP and MATCH for exact match on unsorted data is much faster than ever before when looking up multiple columns (or rows with HLOOKUP) from the same table range.
```
Comment (PrashSMSFT) [inline excel/Concepts/Excel-Performance/excel-tips-for-optimizing-performance-obstructions.md]: Please change "In Excel 16.0.9126.2xxx onwards" to "In O365 version 1809 onwards, Excel's"

File: excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md (hunk 1 above)
Comment (Linda-Editor) [reply excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md]: @PrashSMSFT Let me know when you're done with all your updates, and I'll update the page then. Thanks.

**Comment by PrashSMSFT** [comment]
I added 2 comments on version corrections… LMK if you have any questions.

File: excel/Concepts/Excel-Performance/excel-tips-for-optimizing-performance-obstructions.md (hunk 2 above)
Comment (Linda-Editor) [reply excel/Concepts/Excel-Performance/excel-tips-for-optimizing-performance-obstructions.md]: @PrashSMSFT  Is this the change you want in both topics?

In Office 365 version 1809 and later, Excel's VLOOKUP, HLOOKUP, and MATCH for exact match...

**Pull request merged**